aws/table_aws_s3_bucket.go

```diff
@@ -145,7 +147,7 @@ func tableAwsS3Bucket(_ context.Context) *plugin.Table {
 				Description: "A container for specifying the notification configuration of the bucket. If this element is empty, notifications are turned off for the bucket.",
 				Type:        proto.ColumnType_JSON,
 				Hydrate:     getS3BucketEventNotificationConfigurations,
-				Transform:   transform.FromValue(),
+				Transform:   transform.FromValue().Transform(removeMedataFromS3BucketEventNotificationConfigurations),
```

Comment:
Is this something we should be removing through the transforms, or should we only return the data we want (with one of our own structs) in the `getS3BucketEventNotificationConfigurations` function?

Reply:
What's the advantage of doing it through a transform function, because the code in the transform function almost looks the same (create a struct/map, create properties it in representing those we want from the value, and then returning it)?